### PR TITLE
Add scripts for tests

### PR DESCRIPTION
**Pull request by Jan-M**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 120
+max-line-length = 150
```
Comment (mohabusama) [inline tox.ini]: I suggest we stick to 120 line length.

Comment (Jan-M) [inline tox.ini]: I am annoyed by having to wrap reasonable lines at 30% of my screen width :D

**Comment by Jan-M** [comment]
sorry mixed this push up, this includes also the history function changes.

File: tox.ini (hunk 1 above)
Comment (mohabusama) [inline tox.ini]: Screen sizes are different ;) and if one line breaks the 150 limit we will increase it to 160 :p
imho 120 is actually more than reasonable, also helps consistency over different repos.

**Comment by hjacobs** [comment]
This PR looks outdated, let's close (for now).

**Pull request closed without merging**